monkey/monkey_island/cc/resources/auth/otp.py

```diff
@@ -0,0 +1,23 @@
+from flask import make_response
+
+from monkey_island.cc.flask_utils import AbstractResource
+
+
+class OTP(AbstractResource):
```

Comment:
This should be named `AgentOTP`. OTP's could be used in different ways in the future, no need to assume that we'll only have one use-case for them forever

Reply:
Why would we name this `AgentOTP`? As you said, this "could be used in different ways in the future, no need to assume that we'll only have one use-case for them forever". If we add a new use case, the name of this will be misleading.

These endpoints should not make assumptions about the caller.